src/main/resources/config_file_documentation.txt

```diff
@@ -127,6 +127,14 @@ Config file must be valid JSON format. Examples can be found under the test fold
         value: [0..20]
         default: 4
 
+      "integralWinningThreshold" optional
```

Comment:
I think it would be less confusing to replace all uses of the word "integral" with "integer". I understand after reading the code that you're using "integral" as the adjective form of "integer", but this word has multiple meanings in math and computer science, so just sticking with "integer" makes it a lot more clear (and yes, I also see that there's a precedent for "integral" in the `BigDecimal` API, but I think that was a poor choice on their part).